pkg/cfn/builder/managed_launch_template.go

```diff
@@ -82,11 +83,19 @@ func (m *ManagedNodeGroupResourceSet) makeLaunchTemplateData(ctx context.Context
 			return nil, fmt.Errorf("couldn't build network interfaces for launch template data: %w", err)
 		}
 		if mng.Placement == nil {
-			groupName := m.newResource("NodeGroupPlacementGroup", &gfnec2.PlacementGroup{
-				Strategy: gfnt.NewString("cluster"),
-			})
-			launchTemplateData.Placement = &gfnec2.LaunchTemplate_Placement{
-				GroupName: groupName,
+
+			// Reservations are often explicitly created with placement in mind, and adding
```

Comment:
ok so I got an answer, which was somewhat a surprise (to me) - https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/cr-cpg.html

because the reservation should be created with the placement group, we should change our approach here to NOT create a placement group for reservations. We will accept a placement group (externally created) and use that if one is provided, but we should not automatically create a placement group when a reservation is used

Reply:
yes!